app/src/main/java/com/smlnskgmail/jaman/hashcheckerlite/logic/hashcalculator/ui/HashCalculatorFragment.java

```diff
@@ -353,12 +356,20 @@ private void enterText() {
 
     @Override
     public void appBackClick() {
-        showSnachbarWithAction(
-                getView().findViewById(R.id.fl_main_screen),
-                getString(R.string.message_exit),
-                getString(R.string.action_exit_now),
-                v -> getActivity().finish()
-        );
+        if (doubleBackToExitPressedTwice) {
```

Comment:
Thanks a lot! I'm approve this PR. If you want, you also ca extract time to wait (`2000` ms) to the local static constant (its a best practice to extract all variables that can use as constant to a local fields). This is a "Magic Number", you can read more about this [here](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad).

Reply:
i made magic number, thank you for your advice